mobile/examples/image_classification/android/app/src/main/java/ai/onnxruntime/example/imageclassifier/MainActivity.kt

```diff
@@ -4,6 +4,7 @@
 package ai.onnxruntime.example.imageclassifier
 
 import ai.onnxruntime.*
+import ai.onnxruntime.example.imageclassifier.databinding.ActivityMainBinding
```

Comment:
had to migrate to ViewBinding as the original way is deprecated

https://developer.android.com/topic/libraries/view-binding/migration